saleor/graphql/app/types.py

```diff
@@ -250,6 +255,9 @@ class App(ModelObjectType):
         deprecation_reason=f"{DEPRECATED_IN_3X_FIELD} Use `appUrl` instead.",
     )
     app_url = graphene.String(description="Url to iframe with the app.")
+    manifest_url = graphene.String(
+        description="Url to manifest used during app's installation." + ADDED_IN_35
```

Comment:
Minor thing but on this occasion, I would change "Url" to "URL" here and in other descriptions in this type that are written this way.

Reply:
done